recipes/szip/all/conanfile.py

```diff
@@ -102,17 +100,23 @@ def _create_cmake_module_alias_targets(module_file, targets):
                     set_property(TARGET {alias} PROPERTY INTERFACE_LINK_LIBRARIES {aliased})
                 endif()
             """.format(alias=alias, aliased=aliased))
-        tools.save(module_file, content)
+        save(self, module_file, content)
 
     @property
     def _module_file_rel_path(self):
-        return os.path.join("lib", "cmake", "conan-official-{}-targets.cmake".format(self.name))
+        return os.path.join("lib", "cmake", f"conan-official-{self.name}-targets.cmake")
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "szip")
         self.cpp_info.set_property("cmake_target_name", "szip-shared" if self.options.shared else "szip-static")
-        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.set_property("cmake_target_aliases", ["szip::szip"])
```

Comment:
This does not exist 🙈 

```suggestion
```

Reply:
I don't understand, what doesn't exist?   I want to be able to import szip target as szip::szip ... does this not achieve the goal?